### DISCUSSION TITLE
Please provide docker file for valkey sentinel

### DISCUSSION THREAD

**Discussion by royatanu94** [valkey-io/valkey#768] [Q&A]
Hi,
We used to deploy redis master & sentinel pod with single image but in valkey it seems there are two different image for valkey-master & valkey-sentinel.
I have found valkey docker file : https://github.com/valkey-io/valkey-container
Can you please provide docker file for valkey sentinel
Also please share if you have any doc to migrate valkey-sentinel from redis-sentinel 
TIA

**Comment by edwing-renteria-garibay** [comment]
Hello Royatanu, 

Maybe this will help you.

What I can recommend you is to take a Redis sentinel template and try some configurations against it. It might work, I am new with Valkey hehe.
https://github.com/valkey-io/valkey/blob/unstable/sentinel.conf

If you have a Redis Sentinel template it would be very useful for me to get it.
